javascript/ql/lib/semmle/javascript/security/regexp/RegexTreeView.qll

```diff
@@ -0,0 +1,73 @@
+/**
+ * Provides JavaScript-specific definitions for use in the NfaUtils module.
+ */
+
+private import codeql.regex.nfa.NfaUtils as NfaUtils
+private import codeql.regex.RegexTreeView
+
+/** An implementation that parses a regular expression into a tree of `RegExpTerm`s. */
+module RegexTreeView implements RegexTreeViewSig {
```

Comment:
The lack of consistency between `Regex` and `RegExp` is a bit unfortunate. I understand this differs between languages, but at least in JS-specific code, could we keep it to `RegExp`?